aequilibrae/paths/linear_approximation.py

```diff
@@ -10,12 +11,32 @@
 from aequilibrae.paths.AoN import copy_one_dimension, copy_two_dimensions, copy_three_dimensions
 from aequilibrae import logger
 
+try:
```

Comment:
why not just use scipy.__version__ ??

![image](https://user-images.githubusercontent.com/151124/75393364-a1284400-5939-11ea-8d48-bcdc32808670.png)

Reply:
That was me trying to get around Flake8. But that works as well (did not think it would)

Reply:
#noqa for the win in these situations